src/views/admin/Scanner.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div class="scanner">
+        <Box :full="false" :small="true" class="content back-office" color="white">
+            <div class="row">
+                <div class="col m6 s12">
+                    <div class="input-field">
+                        <input id="id" type="text"
+                               placeholder="enter code using keyboard code scanner"
+                               v-model="inputValue" v-on:change="inputChanged(inputValue)">
```

Comment:
spróbuj watcha zamiast tego:

https://stackoverflow.com/questions/47870889/is-this-a-correct-usage-of-v-onchange